docusaurus/docs/packages/pkg/client/pkg.md

```diff
@@ -113,21 +112,278 @@ txctx -."tx query".-> chain
 go get github.com/pokt-network/poktroll/pkg/client
 ```
 
-## Usage
+## Usage (Supplier Client)
```

Comment:
A thought on all these usages - I think they need comments like you added to the diagrams.

For the supplier client what exactly is this example doing for example?

For the tx client - what are we doing in the example?

Some explanatory comments in the _example_test.go files will clarify the goal of these examples, I think this would help the reader understand what they accomplish with this code.